changelog/pending/20240515--sdk-python--relax-output-all-types-to-better-match-the-implementation.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
```

Comment:
Has this been added accidentally?

Reply:
No, there are two changes here.

The initial change changed all `T_co` uses on static methods to `U`.

The 2nd change changes removes the type variable for from `all` and uses `Any`.